project/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -131,7 +131,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     for (String positionName : distinctPositions) {
       int startIndex = candidatePositions.indexOf(positionName);
       int endIndex = candidatePositions.lastIndexOf(positionName);
```

Comment:
This assumes that all candidates with the same position will be consecutive, right? Maybe state that assumption somewhere.

Reply:
Will do!